committers.md

```diff
@@ -65,6 +65,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Matthew Cheah|Palantir|
```

Comment:
Maybe not a big deal but I thought we keep this list in an order of the last name.

Reply:
Fixed, thanks!